pkg/pxc/app/statefulset/haproxy.go

```diff
@@ -86,12 +86,6 @@ func (c *HAProxy) AppContainer(spec *api.PodSpec, secrets string, cr *api.Percon
 				Name:  "PXC_SERVICE",
 				Value: c.labels["app.kubernetes.io/instance"] + "-" + "pxc",
 			},
-			{
-				Name: "MONITOR_PASSWORD",
```

Comment:
it should be in place if the cr version less than 1.7.0

Reply:
@nonemax  please fix all tests too